.github/workflows/build.yml

```
@@ -1,438 +1,13 @@
-            fi
-            zip -j "mcu-binaries/${sketch_name}_bin-$(date +%Y-%m-%d).zip" "${files[@]}"
-          done
-          ls -la mcu-binaries
-
-      - name: Upload MCU binary artifacts (${{ matrix.ino_dir }})
-        uses: actions/upload-artifact@v6
-        with:
-          name: mcu-binaries-${{ matrix.ino_dir }}
-          path: mcu-binaries/*.zip
-          if-no-files-found: error
+      - run: echo "PR #356 period-statistics confirmation runs in ou-validation"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)